www/data/docs_sidebar.yml

```diff
@@ -1,122 +1,195 @@
+        link: "#hab-origin-key-export"
+      - title: hab origin key generate
+        link: "#hab-origin-key-generate"
+      - title: hab origin key import
+        link: "#hab-origin-key-import"
+      - title: hab origin key upload
+        link: "#hab-origin-key-upload"
+      - title: hab pkg binlink
+        link: "#hab-pkg-binlink"
+      - title: hab pkg build
+        link: "#hab-pkg-build"
+      - title: hab pkg demote
+        link: "#hab-pkg-demote"
+      - title: hab pkg exec
+        link: "#hab-pkg-exec"
+      - title: hab pkg export
+        link: "#hab-pkg-export"
+      - title: hab pkg hash
+        link: "#hab-pkg-hash"
+      - title: hab pkg install
+        link: "#hab-pkg-install"
+      - title: hab pkg path
+        link: "#hab-pkg-path"
+      - title: hab pkg promote
+        link: "#hab-pkg-promote"
+      - title: hab pkg provides
+        link: "#hab-pkg-provides"
+      - title: hab pkg sign
+        link: "#hab-pkg-sign"
+      - title: hab pkg upload
+        link: "#hab-pkg-upload"
+      - title: hab pkg verify
+        link: "#hab-pkg-verify"
+      - title: hab plan init
+        link: "#hab-plan-ini"
+      - title: hab ring key export
+        link: "#hab-ring-key-export"
+      - title: hab ring key generate
+        link: "#hab-ring-key-generate"
+      - title: hab ring key import
+        link: "#hab-ring-key-import"
+      - title: hab svc key generate
+        link: "#hab-service-key-generate"
+      - title: hab studio
+        link: "#hab-studio"
+      - title: hab sup
+        link: "#hab-sup"
+      - title: hab user key generate
+        link: "hab-user-key-generate"
+  - title: Best Practice Guides <span>&plus;</span>
+    link: "/docs/best-practices"
+    sub_links:
+    - title: Quickstart Guide
+      link: "#quickstart-guide"
+    - title: Container Orchestration
+      link: "#container-orchestration"
+    - title: Kubernetes
+      link: "#kubernetes"
+    - title: "ECS and Habitat"
+      link: "#ecs-and-habitat"   
+    - title: "DCOS and Apache Mesos"
+      link: "#mesos-dcos"
+    - title: Core Contribution
+      link: "#core-development"
+    - title: Advanced Plan.sh Writing
+      link: "#planwriting-guide"
+    - title: Iterative Development
+      link: "#iterative-development"
+    - title: Continuous Deployment
+      link: "#continuous-deployment"
+    - title: "Binary Wrapper Packages"  
+      link: "#binary-wrapper"
+  - title: Reference <span>&plus;</span>
+    link: "/docs/reference"
+    sub_links:
+    - title: Environment Variables
+      link: "#environment-variables"
+    - title: Plan Settings
+      link: "#plan-settings"
+    - title: Plan Variables
+      link: "#plan-variables"
+    - title: Plan Helper Functions
+      link: "#utility-functions"
+    - title: Callbacks
+      link: "#reference-callbacks"
+    - title: Hooks
+      link: "#reference-hooks"
+    - title: Mustache Helpers
+      link: "#mustache-helpers"
+    - title: Runtime Settings
```

Comment:
Can we think of a new name

Reply:
Yeah we need to, this is a holdover and I couldnt decide what to call it but this is super misleading.